platform/qt/include/qmapboxgl.hpp

```diff
@@ -111,7 +111,8 @@ class Q_DECL_EXPORT QMapboxGL : public QObject
         MapChangeDidFinishRenderingFrameFullyRendered,
         MapChangeWillStartRenderingMap,
         MapChangeDidFinishRenderingMap,
-        MapChangeDidFinishRenderingMapFullyRendered
+        MapChangeDidFinishRenderingMapFullyRendered,
+        MapChangeSourceAttributionDidChange
```

Comment:
This code will fail on the `static_assert` because we don't have `MapChangeDidFinishLoadingStyle` before `MapChangeSourceAttributionDidChange`.